lumicks/pylake/population/dwelltime.py

```diff
@@ -761,6 +768,60 @@ def label_maker(a, t, n):
         plt.tight_layout()
 
 
+def _exponential_mixture_log_likelihood_gradient(
+    params, t, min_observation_time, max_observation_time
+):
+    """Calculate the gradient of the exponential mixture model.
+
+    Parameters
+    ----------
+    params : array_like
+        array of model parameters (amplitude and lifetime per component)
+    t : array_like
+        dwelltime observations in seconds
+    min_observation_time : float
+        minimum observation time in seconds
+    max_observation_time : float
+        maximum observation time in seconds
+    """
+    amplitudes, lifetimes = np.reshape(params, (2, -1))
+    # We clip the amplitude lower bound. An amplitude of 1e-14 is negligible.
+    amplitudes = np.clip(amplitudes[:, np.newaxis], 1e-14, np.inf)
+    lifetimes = lifetimes[:, np.newaxis]
+    t = t[np.newaxis, :]
+
+    t_min, t_max = min_observation_time, max_observation_time
```

Comment:
Based on my comment below, I think we could just use the short names as the actual arguments (here and in the other internal functions) so we don't end up bloating the code with statements like this. I think the names are still descriptive enough, and we can keep the more verbose names in the public constructor.